client/src/App.js

```diff
@@ -1,16 +1,24 @@
 import { Route, Switch } from "react-router-dom";
 import Login from "./pages/Login";
 import Home from "./pages/Home";
+import Calendar from "./pages/Calendar";
+import NavbarComponent from "./Component/NavbarComponent";
 
 const App = () => (
-	<Switch>
-		<Route path="/" exact>
-			<Home />
-		</Route>
-		<Route path="/login">
-			<Login />
-		</Route>
-	</Switch>
+	<Route>
```

Comment:
```suggestion
	<>
```
We're not using `Route` here for setting a component to show for a path, so it doesn't make sense to use it here. Let's use a React fragment instead

Reply:
OK Coung I will do that.